shared/common-adapters/markdown/generate-emoji-parser.js

```diff
@@ -69,16 +92,22 @@ function genEmojiData() {
 
   emojiLiterals.sort((a, b) => b.length - a.length)
 
-  return {emojiCharacters, emojiIndexByChar, emojiIndexByName, emojiLiterals}
+  return {emojiIndexByChar, emojiIndexByName, emojiLiterals}
 }
 
 function buildEmojiFile() {
-  const p = path.join(__dirname, '..', 'markdown', 'emoji.js')
-  const {emojiLiterals, emojiIndexByName} = genEmojiData()
+  const p = path.join(__dirname, 'emoji-gen.js')
+  const {emojiLiterals, emojiIndexByName, emojiIndexByChar} = genEmojiData()
   const data = `// @noflow
+/* eslint-disable */
 export const emojiRegex = /^(${emojiLiterals.join('|')}|${Object.keys(emojiIndexByName)
     .map(escapeRegExp)
-    .join('|')})/`
+    .join('|')})/
+export const emojiIndexByName = ${JSON.stringify(emojiIndexByName)}
```

Comment:
we were referencing these from the leftover generated stuff (which we turned off) so this brings it back